webpack.common.js

```diff
@@ -49,7 +49,11 @@ var config = {
       },
       {
         test: /\.(png|woff|woff2|eot|ttf|svg).*$/,
-        loader: 'url-loader?limit=100000'
+        loader: 'url-loader',
+        options: {
+            'limit': 10000,
+            'name': 'assets/[hash].[ext]'
```

Comment:
I'm curious if assets that aren't inlined will actually work for our use case. If we add a script tag to a page that pulls in the bundled JS from a CDN, and something in the JS bundle references one of the non-inlined assets, webpack will need to be smart enough to turn that into a url relative to the source url of the script on the CDN -- not the url of the current page -- which it only knows at runtime, so it would need to generate some JS to figure that out. I'd be impressed if it does that.